src/infrastructure/channels/reliable_channel.rs

```diff
@@ -141,9 +141,19 @@ impl Channel for ReliableChannel {
 
         self.dropped_packets = dropped_packets.into_iter().map(|(_, p)| p).collect();
 
-        let a = buffer.len();
-        let b = acked_header.size();
-        // TODO: resent packets if there are dropped packets
         Ok(&buffer[acked_header.size() as usize .. buffer.len()])
     }
+}
+
+impl ReliableChannel {
```

Comment:
If this is something we want to do consistently, please put it in the style guide.

Reply:
More of a suggestion

Reply:
For a code review, let's keep one-off style suggestions to some other venue so they don't clutter things.

Reply:
Or at the very least, mark them as a suggestion explicitly in the PR, and then if the person doesn't follow it, ignore it.

Reply:
Well, I find it easier to get that change in here and do multiple passes of the PR. If we decide that it's not needed etc the comment can be resolved manually.

if we do the change after there is a really high chance it will be forgotten.

Reply:
So it isn't more of a suggestion? =)

Reply:
No, I presented my reason, if he has a reason that is better than we can ignore it. Its more of to start a conversation about it, instead of leaving it there. You are not required to fix every issue requested in a `request changes` review.

Reply:
Saying "get that change in here" implies that it isn't just a suggestion, it is a change you want to see happen. So, style guide. If it is just a suggestion along the lines of "I find it easier if I do...", once is enough. The person will like it and follow it, or not.

Reply:
We have a lot of comments in our PRs that turn into long style discussions, and yet we haven't codified and enforced any that I know of. Either we don't care enough to do the work to do this properly and we can stop making comments about them and accept them, or do we do care enough to do the work to ensure a common set of rules and automation.

Reply:
I would say if that's the still we are maintaining let's do that. I'll update the code and update also our style guide dock.